tests/PHPStan/Analyser/AnalyserIntegrationTest.php

```diff
@@ -802,8 +802,6 @@ public function testBug7094(): void
 
 		$this->assertSame('Parameter #1 $attr of method Bug7094\Foo::setAttributes() expects array{foo?: string, bar?: 5|6|7, baz?: bool}, non-empty-array<string, 5|6|7|bool|string> given.', $errors[5]->getMessage());
 		$this->assertSame(29, $errors[5]->getLine());
-		$this->assertSame('Parameter #1 $attr of method Bug7094\Foo::setAttributes() expects array{foo?: string, bar?: 5|6|7, baz?: bool}, array<\'bar\'|\'baz\'|\'foo\', 5|6|7|bool|string> given.', $errors[6]->getMessage());
-		$this->assertSame(49, $errors[6]->getLine());
```

Comment:
the extracted case is basically https://phpstan.org/r/4ca3ec8a-9f81-4b87-a321-c62a7e58b768
this should be fine IMO